styles/style.css

```diff
@@ -0,0 +1,72 @@
+.body {
+  position: absolute;
+  width: 1440px;
+  height: 569px;
+  left: 0;
+  top: 900px;
+  background-color: #ffffff;
+  color: #383838;
+}
+.h1 {
+  position: absolute;
+  width: 306px;
+  height: 47px;
+  left: 567px;
+  top: 120px;
+}
+.stroke {
+  position: relative;
+  left: 70px;
+  display: none;
+}
+.h1:hover .stroke{
+  display: flex;
+}
+.support {
+  position: absolute;
+  width: 141px;
+  height: 28px;
+  left: 181px;
+  top: 419px;
+}
+.vector3 {
+  position: absolute;
+  left: 190px;
+  top: 275px;
+}
+.communication {
+  position: absolute;
+  width: 296px;
+  height: 28px;
+  left: 416px;
+  top: 419px;
+}
+.vector2 {
+  position: absolute;
+  left: 450px;
+  top: 279px;
+}
+.book {
+  position: absolute;
```

Comment:
remove this method and your page will ha a more readable position

Reply:
If I delete position method every element of my page is moving out of their place, I don't quite understand how to implement it

Reply:
Sorry. I was confused with the number of the string. You can remove position absolute from second string